assert/utils.go

```diff
@@ -2145,3 +2145,21 @@ func formatMapNotContainValueError(target interface{}, mapValue interface{}) str
 
 	return msg.String()
 }
+
+func formatRangeError[T Ordered](actual, min, max T) string {
```

Comment:
Same about minValue, maxValue here


```suggestion
func formatRangeError[T Ordered](actual, minValue, maxValue T) string {
```